opm/test_util/summaryComparator.hpp

```diff
@@ -26,6 +26,17 @@
 #include <algorithm>
 #include <string>
 
+
+// helper macro to handle error throws or not
+#define THROW_OR_NOT(type, message) \
```

Comment:
I would suggest `#define ERROR` - the same seems to indicate that the `THROW_OR_NOT` macro can answer the question: "Should I throw?".